doc/src/dump_image.rst

```diff
@@ -120,12 +120,10 @@ Here are two sample images, rendered as 1024x1024 JPEG files.
 
    Click to see the full-size images:
 
-.. |dump1| image:: JPG/dump1_small.jpg
-   :target: JPG/dump1.jpg
-   :width: 48%
-.. |dump2| image:: JPG/dump2_small.jpg
-   :target: JPG/dump2.jpg
-   :width: 48%
+.. |dump1| image:: img/dump1.jpg
+   :scale: 48%
```

Comment:
i don't think this is a correct change. you still want the images to be scaled to 48% of the width of the "screen" not 48% of its original size.